public/locales/de/common.json

```
@@ -5,5 +5,13 @@
   },
   "StartExploring": {
     "button": "Beginne mit der Erkundung"
+  },
+  "statisticsDisplay": {
+    "prefix": "Über ",
```

let's not handle space in translation
```suggestion
    "prefix": "Über",
```